src/components/note/note.js

```diff
@@ -1,60 +1,98 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import BookImage from '../book-image/book-image';
+import Icon from '@mapbox/mr-ui/icon';
+
+class Image extends React.Component {
+  render() {
+    const { color, icon } = this.props;
+    return (
+      <div
+        className={`bg-${color}-light round-full color-${color}-dark flex-parent flex-parent--center-main flex-parent--center-cross`}
+        style={{ width: 50, height: 50 }}
+      >
+        <Icon name={icon} size="40" />
+      </div>
+    );
+  }
+}
+
+Image.propTypes = {
+  color: PropTypes.string,
+  icon: PropTypes.string
+};
+
 class Note extends React.Component {
   render() {
-    const { props } = this;
-    let titleText = 'Note';
-    const themes = {
-      base: {
-        // applied to every note
-        padding: '18px',
-        fontSize: '13px',
-        lineHeight: '20px',
-        borderRadius: '4px'
-      },
+    const { theme, title, children, image } = this.props;
+    let themes = {
       default: {
         background: '#f1faff',
-        color: '#587594'
+        color: '#587594',
+        image: <BookImage />
       },
       warning: {
         background: '#feefe2',
-        color: '#945823'
+        color: '#945823',
+        image: <Image icon="alert" color="orange" />
       },
       error: {
         background: '#fbe5e5',
-        color: '#ba3b3f'
+        color: '#ba3b3f',
+        image: <Image icon="alert" color="red" />
       },
-      new: {
+      beta: {
         background: '#e8f5ee',
-        color: '#1b7d4f'
+        color: '#1b7d4f',
+        image: <Image icon="marker" color="green" />
+      },
+      download: {
+        background: '#f2effa',
+        color: '#5a3fc0',
+        image: <Image icon="arrow-down" color="purple" />
       }
     };
-    if (props.title) {
-      titleText = props.title;
-    }
-    const themeNote = Object.assign(
-      themes['base'],
-      themes[props.theme] || themes['default']
-    );
+    themes['legacy'] = themes.warning;
+    themes['new'] = themes.beta;
 
     return (
-      <div className="flex-parent flex-parent--row mb18" style={themeNote}>
-        <div className="flex-child mr12">{props.imageComponent}</div>
-        <div className="flex-child">
-          <div className="txt-bold txt-m mb6">{titleText}</div>
-          {props.children}
+      <div
+        className="dr-ui--note py18 px18 round flex-parent flex-parent--row mb18"
+        style={{
+          background: themes[theme].background,
+          color: themes[theme].color
+        }}
+      >
+        {image && <div className="flex-child mr12">{themes[theme].image}</div>}
+        <div className="flex-child prose">
+          <div className="txt-bold txt-m mb6">{title}</div>
+          {children}
         </div>
       </div>
     );
   }
 }
 
+Note.defaultProps = {
+  theme: 'default',
+  title: 'Note',
```

Comment:
What do you think about different default titles for the different themes? Is that unnecessarily complicated? For example `Public Beta` for `beta` and `Download` for `download`.

Reply:
Super smart! Done!

![image](https://user-images.githubusercontent.com/2180540/71285031-31db1600-2332-11ea-9a30-d66a56081e05.png)